CMakeLists.txt

```
@@ -133,6 +133,14 @@ ecbuild_add_option( FEATURE FIELD_API
                     DEFAULT ON
                     REQUIRED_PACKAGES "field_api" )
 
+ecbuild_add_option( FEATURE FIELD_API_ENABLE_IO
+		    DESCRIPTION "Use field_api IO to store field_api objects to disk."
+                    CONDITION HAVE_FIELD_API
+		    DEFAULT OFF )
```

Mixture of tabs and spaces
```suggestion
                    DESCRIPTION "Use field_api IO to store field_api objects to disk."
                    CONDITION HAVE_FIELD_API
                    DEFAULT OFF )
```